web/utils/query.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config(); //Load from the .env file
+const mysql = require('mysql2');
+
+export const pool = mysql.createPool({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_DATABASE,
+    waitForConnections: true,
+    connectionLimit: 10
+})
+
+export async function query(q, values) {
+  try {
+    const results = await pool.query(q, values)
+    await pool.end()
```

Comment:
We actually don't want to ever close this pool. It should always be available for the lifecycle of the server.